lib/common.js

```diff
@@ -922,9 +921,9 @@ module.exports = {
             /*
              * MANTA-331: while a trailing '/' is ok in HTTP, this messes with
              * the consistent hashing, so ensure there isn't one
+             *
+             * This is now handled by the restify plugin sanitizePath
              */
-            /* JSSTYLED */
-            req._path = req._path.replace(/\/*$/, '');
```

Comment:
Perhaps just drop this comment and add a comment where sanitizePath() is used
mentioning its importance and why.

Reply:
agreed